.github/workflows/update-docs-version.yml

```diff
@@ -0,0 +1,30 @@
+name: Update docs version
+
+on:
+    release:
+        types: [published]
+
+jobs:
+    build:
+        runs-on: ubuntu-18.04
+        steps:
+            -   uses: actions/checkout@v1
+            -   name: Update latest_version property in mkdocs.yml
+                run: |
+                    sed -i "s/latest_version: .*/latest_version: ${GITHUB_REF##*/}/g" mkdocs.yml
+            -   name: Commit changes
+                env:
+                    COMMIT_MSG: |
+                        Update documentation version to ${GITHUB_REF##*/}
```

Comment:
note that `${GITHUB_REF##*/}` is bash's syntax, not GHA, and the commit message will not have a version in it, but the placeholder